kvmagent/kvmagent/plugins/nvram/nvram_local.py

```
@@ -248,3 +109,11 @@ def find_nvram_install_path_by_mount_folder(vm_uuid):
             raw_path = f.read().strip()
             return os.path.abspath(raw_path) if raw_path else ''
     return ''
+
+def detach_loop_device(loop_path):
+    # type: (str) -> None
+    if loop_path:
+        if os.path.exists(loop_path):
+            r, o, e = bash.bash_roe("losetup -d %s" % pipes.quote(loop_path))
```

Ruff (0.15.2)

[warning] Unpacked variable `o` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)